packages/scripts/config/webpack.config.js

```diff
@@ -135,7 +136,6 @@ const config = {
 				test: /\.jsx?$/,
 				exclude: /node_modules/,
 				use: [
-					require.resolve( 'thread-loader' ),
```

Comment:
It might be fine to keep `thread-loader` but I removed it temporarily until the integration works. We can enable it back later.

Reply:
I read at https://webpack.js.org/loaders/thread-loader/:
> Each worker is a separate node.js process, which has an overhead of ~600ms. There is also an overhead of inter-process communication.
>
> Use this loader only for expensive operations!

Let's remove this loader and simplify things.